content/vendor/additional-objects/k8s-sig-application.md

```diff
@@ -0,0 +1,8 @@
+---
+date: 2019-11-27
+linktitle: "K8s SIG Application"
+title: K8s SIG Application
+weight: 20030
+---
+
+The [Kubernetes SIG Application](https://github.com/kubernetes-sigs/application#kubernetes-applications) spec provides a standard API for creating, viewing, and managing applications.  A SIG Application spec provides helpful metadata to other Kubernetes tools, and enables important KOTS features such as [links to the deployed application](/vendor/dashboard/open-buttons/#provide-an-application-crd).
```

Comment:
We should list the fields that are required and try to explain how to use this as part of a KOTS application.